recipes/quest/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "quest" %}
+{% set version = "3.0-a1" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/erdc/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - python >3
+    - pip
+  run:
+    - python >3
+
+test:
+  imports:
+    - quest
+    - quest_provider_plugins
+    - quest_io_plugins
+    - quest_tool_plugins
+    - quest.tools
+
+about:
+  home: https://github.com/erdc/quest
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'https://github.com/erdc/quest'
+
+  description: |
+    Quest is a Python library that provides the ability to search, publish and 
+    download data (both geographical and non-geographical) from multiple data 
+    sources, including local repositories and web-based data providers. Quest 
+    also provides a suite of tools for manipulating and transforming data once 
+    it is downloaded.
+  doc_url: https://quest.readthedocs.io/
+  dev_url: https://github.com/erdc/quest
+
+extra:
+  recipe-maintainers:
+    - AaronV77
+    - sdc50
+    - dharhas
```

Comment:
Does everyone agree to be a maintainer ?

cc: @sdc50 @dharhas

Reply:
Agree.

Reply:
yup.

Reply:
Yes